migtests/tests/oracle-tests/ora-hr-db-sanity-test/env.sh

```diff
@@ -0,0 +1,9 @@
+export SOURCE_DB_TYPE="oracle"
+export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"ORCL"}
+export SOURCE_DB_HOST=${SOURCE_DB_HOST:-"10.9.8.213"}
```

Comment:
As suggested in other similar PR, please move setting of host, user, and password out of this env.sh .